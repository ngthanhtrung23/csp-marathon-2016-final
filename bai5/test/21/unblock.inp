.1223.
.1443.
0056..
7.56..
7.8699
7.8AAA
