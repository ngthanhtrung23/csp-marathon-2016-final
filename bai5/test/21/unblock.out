28
3 D 2
2 R 1
4 R 1
5 U 2
0 R 1
7 U 1
8 U 1
A L 3
6 D 1
0 R 1
1 D 3
0 L 1
6 U 1
A R 3
7 D 1
0 L 1
8 D 1
5 D 2
2 L 3
4 L 3
3 U 2
5 U 2
0 R 1
7 U 1
8 U 1
A L 3
6 D 1
0 R 3

